mods/d2k/maps/atreides-06b/atreides06b.lua

```diff
@@ -0,0 +1,259 @@
+
+	-- fix bug where ClosestEdgeCell return wrong cell for bottom/Right corner
+	smugglers = { CPos.New( Map.ClosestEdgeCell(SmugglersUnitSpawn.Location).X - 1, Map.ClosestEdgeCell(SmugglersUnitSpawn.Location).Y - 1), SmugglersUnitSpawn.Location },
+	-- fix bug where ClosestEdgeCell return wrong cell for bottom/Right corner`
+	ordosSmall = { CPos.New( Map.ClosestEdgeCell(OrdosSmallUnitSpawn.Location).X - 1, Map.ClosestEdgeCell(OrdosSmallUnitSpawn.Location).Y), OrdosSmallUnitSpawn.Location },
+	mercenaries = { Map.ClosestEdgeCell(MercenariesUnitSpawn.Location), MercenariesUnitSpawn.Location }
+}
+
+OrdosReinforcementsPaths =
+{
+	{ Map.ClosestEdgeCell(OrdosReinforcements1.Location), OrdosReinforcements1.Location },
+	{ Map.ClosestEdgeCell(OrdosReinforcements2.Location), OrdosReinforcements2.Location },
+	{ Map.ClosestEdgeCell(OrdosReinforcements3.Location), OrdosReinforcements3.Location }
+}
+
+AtreidesReinforcementsPaths =
+{
+	{ Map.ClosestEdgeCell(AtreidesReinforcementsPoint1.Location), AtreidesReinforcementsPoint1.Location},
+	{ Map.ClosestEdgeCell(AtreidesReinforcementsPoint2.Location), AtreidesReinforcementsPoint2.Location}
+
+}
+
+SendAtreidesReinforcements = function(delay, squad, path)
+	Trigger.AfterDelay(delay, function()
+		Reinforcements.ReinforceWithTransport(Atreides, "carryall.reinforce", squad, path, { path[1] })
+		Trigger.AfterDelay(DateTime.Seconds(5), function()
+			Media.PlaySpeechNotification(Atreides, "Reinforce")
+		end)
+	end)
+end
+
+SendStarportReinforcements = function()
+	Trigger.AfterDelay(OrdosStarportDelay[Difficulty], function()
+		if OStarport.IsDead or OStarport.Owner ~= OrdosSmall then
+			return
+		end
+
+		local units = Reinforcements.ReinforceWithTransport(OrdosSmall, "frigate", OrdosStarportReinforcements[Difficulty], { Map.ClosestEdgeCell(OStarport.Location), OStarport.Location + CVec.New(1, 1) }, { Map.ClosestEdgeCell(OStarport.Location) })[2]
+		Utils.Do(units, function(unit)
+			unit.AttackMove(AttackLocation)
+			IdleHunt(unit)
+		end)
+
+		SendStarportReinforcements()
+	end)
+end
+
+Tick = function()
+
+	if Atreides.HasNoRequiredUnits() then
+		Ordos.MarkCompletedObjective(KillAtreides1)
+		OrdosSmall.MarkCompletedObjective(KillAtreides2)
+	end
+
+	if Ordos.HasNoRequiredUnits() and OrdosSmall.HasNoRequiredUnits() and not OrdosKilled then
+		Media.DisplayMessage(UserInterface.GetFluentMessage("ordos-annihilated"), Mentat)
+		OrdosKilled = true
+		Atreides.MarkCompletedObjective(KillOrdos)
+	end
+
+	if Smugglers.HasNoRequiredUnits() and not SmugglersKilled then
+		Media.DisplayMessage(UserInterface.GetFluentMessage("smugglers-annihilated"), Mentat)
+		SmugglersKilled = true
+		Atreides.MarkCompletedObjective(KillSmugglers)
+	end
+
+	if Mercenaries.HasNoRequiredUnits() and not MercenariesKilled then
+		Media.DisplayMessage(UserInterface.GetFluentMessage("smugglers-annihilated"), Mentat)
```

Comment:
```suggestion
		Media.DisplayMessage(UserInterface.GetFluentMessage("mercenaries-annihilated"), Mentat)
```
There is a message from Harkonnen 08 that can be reused here.